src/backend/distributed/sql/citus--12.1-1--13.0-1.sql

```diff
@@ -0,0 +1,3 @@
+-- citus--12.1-1--13.0-1.sql
```

Comment:
On main branch the current version is '12.2-1' as seen in https://github.com/citusdata/citus/blob/665d72a2f57bf2c94241ac9892306ac7c70d098a/src/backend/distributed/citus.control#L3C19-L3C27

Currently you do not have a valid upgrade path from main branch to this release branch.

IMO, we should remove all references to 12.2 as we are skipping that version. I suggest we rename 12.2 to 13.0 on the main branch. After that make sure that similar changes are also applied on this release branch.

I guess, rebasing this release branch is not an option. Instead we can copy all the relevant sql files from the main branch that are relevant for 13.0-1 to this branch.